### PR TITLE
Add custom DAO subtitles for Identity and Meme DAOs

### PR DESCRIPTION
**Pull request by tibetsprague**

### CODE REVIEW

**Comment by tibetsprague** [comment]
Fixes https://daostack.tpondemand.com/RestUI/Board.aspx#page=board/5209716961861964288&appConfig=eyJhY2lkIjoiQjgzMTMzNDczNzlCMUI5QUE0RUE1NUVEOUQyQzdFNkIifQ==&boardPopup=userstory/1029/silent

File: src/components/ViewDao/DaoSidebar.tsx [hunk 1]
```diff
@@ -30,7 +30,17 @@ class DaoSidebarComponent extends React.Component<IProps, null> {
               <b className={css.daoIcon} style={{backgroundImage: bgPattern.toDataUrl()}}></b>
               <span>{dao.name}</span>
             </Link>
-            <p>Anyone can make a proposal to the DAO! Click the button on the top right.</p>
+          </div>
+          <div className={css.daoDescription}>
+            { dao.name === "Meme" ?
```
Comment (jellegerbrandy) [inline src/components/ViewDao/DaoSidebar.tsx]: This is really very very ugly... :-/ we should consider having a registry, liek for the schemes ....

Comment (jellegerbrandy) [reply src/components/ViewDao/DaoSidebar.tsx]: let;'s merge it anyway and backlog https://github.com/daostack/alchemy/issues/751

Comment (tibetsprague) [reply src/components/ViewDao/DaoSidebar.tsx]: yes of course, I thought we are already planning a big team conversation about a DAO registry

**Pull request merged**